release-notes/azure/v11.3.0.md

```diff
@@ -0,0 +1,10 @@
+## :zap:  Giant Swarm Release 11.3.0 for Azure :zap:
+
+This release replaces CoreOS Container Linux that is going [end of life](https://coreos.com/os/eol/) on 26th of May 2020 with Flatcar Container Linux. 
+
+Migration process requires replacing VMSS CoreOS image with Flatcar image which, [due to Azure limitations](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-upgrade-scale-set#create-time-properties), is not possible. In order to enable existing clusters to upgrade with this release we had to amend standard procedure with additional steps that replace old VMSS with a new one, ensuring all workload is moved. 
+
+### Warning
+Due to additional steps in upgrade process, the whole procedure will take more time, up to a few hours. It also requires manual steps on our side, so please **schedule all upgrades with your SE at least one day in advance**. This will allow us to prepare and guarantee appropriate support throughout the upgrade process.
+
+Initial tests shown up to 2% loss of incoming traffic over the time period of 5-10min. It was not confirmed in the following tests but because of the complexity of the process you should be aware that this may occur.
```

Comment:
is this still true? During standup I understood we had 100% uptime, but I may have been distracted doing memes.

Reply:
I think after fixing @whites11 still observed 2% drop which was better than first 25% but it is correct that @tuommaki had 100% uptime. I wasn't sure if it was something you changed @tuommaki or your tests didn't show any downtime?

Reply:
Let's not say it is 100% uptime as they will complain on that for sure :D